perceval/backends/core/confluence.py

```diff
@@ -215,7 +227,7 @@ def parse_historical_content(raw_json):
     def _init_client(self, from_archive=False):
         """Init client"""
 
-        return ConfluenceClient(self.url, archive=self.archive, from_archive=from_archive)
+        return ConfluenceClient(self.url, self.add_ancestors, archive=self.archive, from_archive=from_archive)
```

Comment:
in order to ensure that the data returned by Perceval is consistent, it's better to include the ancestors info as something that is always returned by Perceval. What do you think about adding a method (`ancestors`) to the client to retrieve the ancestor pages ?

Reply:
Yep, makes sense, I'll change that